src/helpers/copyContract.js

```diff
@@ -0,0 +1,26 @@
+export default async function copyContract(contractClass, address, txParams = {}) {
+  address = address.replace('0x', '')
+
+  // This is EVM assembly will return of the code of a foreign address.
+  //
+  // operation    | bytecode   | stack representation
+  // =================================================
+  // push20 ADDR  | 0x73 ADDR  | ADDR
+  // dup1         | 0x80       | ADDR ADDR
+  // extcodesize  | 0x3B       | ADDR 0xCS
+  // dup1         | 0x80       | ADDR 0xCS 0xCS
+  // swap2        | 0x91       | 0xCS 0xCS ADDR
+  // push1 00     | 0x60 0x00  | 0xCS 0xCS ADDR 0x00
+  // dup1         | 0x80       | 0xCS 0xCS ADDR 0x00 0x00
+  // swap2        | 0x91       | 0xCS 0xCS 0x00 0x00 ADDR
+  // extcodecopy  | 0x3C       | 0xCS
+  // push1 00     | 0x60 0x00  | 0xCS 0x00
+  // return       | 0xF3       |
+
+  const ASM_CODE_COPY = `0x73${address}803b8091600080913c6000f3`
+
+  const params = Object.assign({}, contractClass.defaults(), txParams, { to: 0x0, data: ASM_CODE_COPY })
```

Comment:
It seems that sending a transaction to `0x0` (in geth and parity) does not actually create a new contract, but fails because it is a malformed address (ganache is happy to accept it). Sending to the `0x0....0` address is accepted in geth and parity, but actually sends a tx to that address (instead of creating a contract); ganache instead fails because it is not a valid contract address (same with any other address).

In order to actually create a contract (in all nodes), one needs to send the tx to `null` instead of `0x0`. Yet another point for testing on geth instead of ganache.